org.adempiere.base/src/org/compiere/util/DefaultEvaluatee.java

```diff
@@ -104,15 +104,20 @@ public DefaultEvaluatee(PO po) {
 		this(new PODataProvider(po));
 	}
 	
+	/**
+	 * @param dataProvider
+	 */
+	public DefaultEvaluatee(DataProvider dataProvider, int windowNo, int tabNo) {
+		this(dataProvider);
+		this.m_windowNo = windowNo;
+		this.m_tabNo = tabNo;
+	}
+	
 	/**
 	 * @param dataProvider
 	 */
 	public DefaultEvaluatee(DataProvider dataProvider) {
-		this.m_dataProvider = dataProvider;
-		this.m_windowNo = 0;
-		this.m_tabNo = -1;
-		this.m_onlyWindow = false;
-		this.m_onlyTab = null;
+		
```

Comment:
It is fine to add a new constructor but why the need to remove the above ?

Reply:
Sorry, I saw it, but I thought it was collapsed code.